warehouse/forklift/legacy.py

```diff
@@ -909,7 +911,11 @@ def file_upload(request):
             raise _exc_with_message(exc.__class__, exc.detail) from None
 
         project_service = request.find_service(IProjectService)
-        project = project_service.create_project(form.name.data, request.user)
+        try:
+            project = project_service.create_project(form.name.data, request.user)
+        except RateLimiterException:
+            msg = "Too many new projects created"
+            raise _exc_with_message(HTTPTooManyRequests, msg)
```

Comment:
We probably need to do something similar in the other places where `create_project` is used.

Reply:
That works too!

Reply:
see https://github.com/pypi/warehouse/pull/13097/commits/42693eb63f627611e59970886f345d7669f95b47